composer.json

```diff
@@ -10,7 +10,7 @@
     "aws/aws-sdk-php-laravel": "^3.1",
     "barryvdh/laravel-debugbar": "^2.3",
     "doctrine/dbal": "~2.5.13",
-    "dosomething/gateway": "^1.7.0",
+    "dosomething/gateway": "dev-test-helpers",
```

Comment:
is this a stable version?

Reply:
I'll run a `composer update` with the stable release as soon as DoSomething/gateway#97 is approved!

Reply:
Updated to use stable release in 045d5ff!